setup.js

```diff
@@ -0,0 +1,6 @@
+const { writeFile } = require("fs/promises")
+
+writeFile(".env.test", "PGDATABASE=nc_news_test")
+writeFile(".env.development", "PGDATABASE=nc_news")
+writeFile("query.sql", "\\c nc_news;")
+
```

Comment:
Completely fine to use things like to this to aid our development process, but it's probably worth getting rid of them once we've used them!

Reply:
Agreed to leave in